nengo_bones/scripts/check_bones.py

```diff
@@ -12,6 +12,74 @@
 from nengo_bones.templates import BonesTemplate, load_env
 
 
+def _check_file(filename, *, config, env, path, verbose):
+    full_filename = filename.replace("pkg", config["pkg_name"])
+    click.echo(full_filename + ":")
+
+    # TODO: Ensure that the file is there <=> it is in the config
+    if not (path / full_filename).exists():
+        click.echo("  File not found")
+        return True
+
+    with (path / full_filename).open(encoding="utf-8") as f:
+        current_lines = f.readlines()
+
+    for line in current_lines[:50]:
+        if "Automatically generated by nengo-bones" in line:
+            break
+    else:
+        click.echo("  This file was not generated with nengo-bones")
+        return True
+
+    template = BonesTemplate(filename, env)
+    if template.section not in config:
+        click.secho(
+            "  This file contains 'Automatically generated by nengo-bones',\n"
+            "  but there is no corresponding configuration in .nengobones.yml.\n"
+            "  Please remove this text or configure it in .nengobones.yml.",
+            fg="red",
+        )
+        return False
+
+    new_lines = template.render(**template.get_render_data(config)).splitlines(
+        keepends=True
+    )
+
+    diff = list(
+        difflib.unified_diff(
+            current_lines,
+            new_lines,
+            fromfile=f"current {filename}",
+            tofile=f"new {filename}",
+        )
+    )
+
+    if (
+        "dev = 0\n" in current_lines
+        and filename == "pkg/version.py"
+        and len(diff) == 11
+        and diff[6].startswith("-version_info")
```

Comment:
What about if instead of having this special case code for the version, we add something like a `# bones: ignore` in-line flag? That would allow for a simpler check here, and might be generally useful in other circumstances.